subscription/src/test/java/org/killbill/billing/subscription/api/user/TestUserApiExpire.java

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright 2010-2022 Ning, Inc.
```

Comment:
FYI: You can drop copyright for `Ning`, `Groupon` on all new files.

Reply:
Fixed